### PR TITLE
Adjust setup-miniconda parameters

### PR DESCRIPTION
**Pull request by maresb**
This is in hopes of resolving sporadic errors.
Ref: <https://github.com/conda-incubator/setup-miniconda/issues/274>

### Motivation for these changes
...

### Implementation details
...

### Checklist
+ [ ] Explain motivation and implementation 👆
+ [ ] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+ [ ] Link relevant issues, preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+ [ ] The commits correspond to [_relevant logical changes_](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes). Note that if they don't, we will [rewrite/rebase/squash the git history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_rewriting_history) before merging.
+ [ ] Are the changes covered by tests and docstrings?
+ [ ] Fill out the short summary sections 👇

## Major / Breaking Changes
- ...

## New features
- ...

## Bugfixes
- ...

## Documentation
- ...

## Maintenance
- ...

### CODE REVIEW

**Comment by maresb** [comment]
I find the "Benchmark" test unconvincing.

**Comment by ricardoV94** [comment]
> I find the "Benchmark" test unconvincing.

What do you mean?

**Comment by maresb** [comment]
It seems to often fail for no apparent reason. Like here.

**Comment by ricardoV94** [comment]
The failure seems exactly like the one for the other jobs before this PR. Perhaps you also need to tweak here? https://github.com/pymc-devs/pytensor/blob/0b272c655085431c5501a786908b17456b6a4e04/.github/workflows/test.yml#L167-L170

**Comment by ferrine** [comment]
It seems to boil down to dependencies resolution

https://github.com/pymc-devs/pytensor/actions/runs/3910005116/jobs/6681731220#step:3:544

**Comment by maresb** [comment]
Ah, thanks! I didn't look closely enough. Hopefully that does it.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -164,11 +164,9 @@ jobs:
...
           uses: conda-incubator/setup-miniconda@v2
           with:
-            mamba-version: "*"
-            channels: conda-forge,defaults
-            channel-priority: true
-            python-version: 3.9
-            auto-update-conda: true
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            python-version: ${{ matrix.python-version }}
```
Comment (ricardoV94) [inline .github/workflows/test.yml]: Shouldn't it be fixed? We don't want to run benchmarks in all versions

**Review by ricardoV94 — APPROVED** [review]
Looks like it's working!

**Comment by ricardoV94** [comment]
Thanks @maresb! Hopefully it's stable now :)

**Pull request merged**